components/molecule/radioButtonField/src/index.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import MoleculeField from '@s-ui/react-molecule-field'
+import AtomRadioButton from '@s-ui/react-atom-radio-button'
+
+const BASE_CLASS = 'sui-MoleculeRadioButtonField'
+
+const getErrorState = (success, error) => {
+  if (success) return false
+  if (error) return true
+}
+
+const MoleculeRadioButtonField = ({
+  id,
+  label,
+  successText,
+  errorText,
+  helpText,
+  onChange,
+  ...props
+}) => {
+  const errorState = getErrorState(successText, errorText)
+  return (
+    <div className={BASE_CLASS}>
```

Comment:
is this div really needed, couldn't the moleculeField be the wrapper element?

Reply:
I think, that, in order to apply specific styles to elements inside this component, the right way to do it is to assign a specific class to this component, and we need a `div` to do that as we cannot assign a className directly to a `MoleculeField` component